python/doc/demos/NODDI_turorial_01.md

```diff
@@ -0,0 +1,124 @@
+# Tutorial for python-AMICO/NODDI
+
+Here you can learn how to run python-AMICO/NODII for sampel dateset.
+
+## 1. Prepare dataset
+
+The sample dataset is available from [NODDI official website](http://mig.cs.ucl.ac.uk/index.php?n=Tutorial.NODDImatlab).
+
+After unzip, make the directory strucutre as followings:
+
+```
+└── Study01
+    └── Subject01
+        ├── NODDI_DWI.hdr
+        ├── NODDI_DWI.img
+        ├── NODDI_protocol.bval
+        ├── NODDI_protocol.bvec
+        ├── brain_mask.hdr
+        ├── brain_mask.img
+        ├── roi_mask.hdr
+        └── roi_mask.img
+```
+
+## 2. DWI preprocess
+
+Usually you need some preprocesses for DWI images (e.g., eddy current correction, head movement correction, and skull stripping).
+You need do these preprocesses before performing python-AMICO/NODDI.
```

Comment:
`[MOD]` "preprocesses" --> "preprocessing"
`[MOD]` "performing python-AMICO/NODDI" --> "fitting the model"